openedx/adg/lms/applications/management/commands/update_is_bu_prerequisite_courses_passed.py

```diff
@@ -0,0 +1,94 @@
+"""
+Management command to update is_bu_prerequisite_courses_passed flag in ApplicationHub
+"""
+import logging
+import sys
+
+from django.contrib.auth.models import User
+from django.core.management.base import BaseCommand
+
+from openedx.adg.lms.applications.helpers import (
+    check_user_passed_given_courses,
+    get_users_with_active_enrollments_from_course_groups,
+    send_application_submission_confirmation_email
+)
+from openedx.adg.lms.applications.models import ApplicationHub, MultilingualCourseGroup
+
+logger = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
+    """
+    Updates is_bu_prerequisite_courses_passed flag in ApplicationHub.
+    """
+
+    def handle(self, *args, **kwargs):  # pylint: disable=unused-argument
+        bu_course_groups = MultilingualCourseGroup.objects.business_line_and_common_business_line_prereq_course_groups()
+
+        if not bu_course_groups:
+            sys.exit('Exiting!!! No business line or common business line pre-req course found.')
+
+        users_to_be_checked_for_update = self.get_users_with_bu_pre_reqs_not_marked_as_passed()
+        users_to_be_checked_for_update = get_users_with_active_enrollments_from_course_groups(
+            users_to_be_checked_for_update, bu_course_groups
+        )
```

Comment:
We may write this as 
```
users_to_be_checked_for_update = get_users_with_active_enrollments_from_course_groups(
    self.get_users_with_bu_pre_reqs_not_marked_as_passed(), 
    bu_course_groups
)
```

Reply:
We can write it this way as well but for readability the above method is preferable. And there is no additional cost.